src/Components/Blazor/Templates/src/content/BlazorHosted-CSharp/BlazorHosted-CSharp.Client/App.razor

```diff
@@ -1 +1,5 @@
-﻿<Router AppAssembly="typeof(Program).Assembly" />
+﻿<Router AppAssembly="typeof(Program).Assembly">
```

Comment:
This is because this component is route-able, but doesn't really have any meaning?

Reply:
The `AppAssembly` parameter specifies which assembly to look in to find the routable page components, and it's not necessarily the same as your app entrypoint assembly.